src/pigui/Makefile.am

```diff
@@ -0,0 +1,20 @@
+include $(top_srcdir)/Makefile.common
```

Comment:
Btw, please remove all autotools stuff, as suport for that was dropped yesterday.

Also squash commits?

Reply:
Dropped the autotools stuff.

Turns out squashing wasn't the issue, I must have made a mistake rebasing, so I redid it. The first 2 commits that were here originally are now identical to master.

Reply:
Ok, now it should be fine